src/devices/DCMotor/DCMotor.cs

```diff
@@ -32,67 +34,185 @@ protected DCMotor(GpioController controller)
+        /// <paramref name="directionPin"/> should be connected to H-bridge input corresponding to one of the motor inputs.
+        /// Connecting motor directly to GPIO pin is not recommended and may damage your board.
+        /// </remarks>
+        public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, IGpioController controller = null)
+        {
+            if (speedControlChannel == null)
+                throw new ArgumentNullException(nameof(speedControlChannel));
+
+            if (directionPin == -1)
+                throw new ArgumentOutOfRangeException(nameof(directionPin));
+
+            return new DCMotor2PinNoEnable(speedControlChannel, directionPin, controller);
+        }
+        
+        /// <summary>
+        /// Creates <see cref="DCMotor"/> instance which allows to control speed in both directions.
+        /// </summary>
+        /// <param name="speedControlPin">Pin used to control the speed of the motor with software PWM (frequency will default to 50Hz)</param>
+        /// <param name="directionPin">Pin used to control the direction of the motor</param>
+        /// <param name="controller">GPIO controller related to <paramref name="speedControlPin"/> and <paramref name="directionPin"/></param>
+        /// <returns><see cref="DCMotor"/> instance</returns>
+        /// <remarks>
+        /// PWM pin <paramref name="speedControlPin"/> can be connected to either enable pin of the H-bridge.
+        /// or directly to the input related with the motor (if H-bridge allows inputs to change frequently).
+        /// <paramref name="directionPin"/> should be connected to H-bridge input corresponding to one of the motor inputs.
+        /// Connecting motor directly to GPIO pin is not recommended and may damage your board.
+        /// </remarks>
+        public static DCMotor Create(int speedControlPin, int directionPin, IGpioController controller = null)
+        {
+            if (speedControlPin == -1)
+                throw new ArgumentOutOfRangeException(nameof(speedControlPin));
+
+            if (directionPin == -1)
+                throw new ArgumentOutOfRangeException(nameof(directionPin));
+
+            controller = controller ?? new GpioController();
+            return new DCMotor2PinNoEnable(
+                new SoftwarePwmChannel(speedControlPin, DefaultPwmFrequency, 0.0, controller: controller),
+                directionPin,
+                controller);
+        }
+
+        /// <summary>
+        /// Creates <see cref="DCMotor"/> instance which allows to control speed in both directions.
+        /// </summary>
+        /// <param name="speedControlChannel"><see cref="PwmChannel"/> used to control the speed of the motor</param>
+        /// <param name="directionPin">Pin used to control the direction of the motor</param>
+        /// <param name="otherDirectionPin">Pin used to control the direction of the motor</param>
```

Comment:
Honestly I still think it is not super clear to me why to pick the dual direction pin vs the single pin direction one until I read in the remarks section. Perhaps we should add some of that info into the Summary? Anyway, this may also be because I haven't connected one of this before so it might just be that I'm not used to using these devices.

Reply:
We can improve that once we get someone using it and complaining about summary/remarks - it's hard to pick easier words after you already know how something works